flexmeasures/data/schemas/generic_assets.py

```diff
@@ -46,29 +46,6 @@ class GenericAssetSchema(ma.SQLAlchemySchema):
     class Meta:
         model = GenericAsset
 
-    @validates_schema(skip_on_field_errors=False)
-    def validate_name_is_unique_in_account(self, data, **kwargs):
```

Comment:
We should replace this with a uniqueness check on the child's name with respect to its parent.